packages/ui/src/partials/w3m-core-button/index.ts

```diff
@@ -8,14 +8,15 @@ import '../w3m-connect-button'
 @customElement('w3m-core-button')
 export class W3mCoreButton extends LitElement {
   // -- state & properties ------------------------------------------- //
-  @state() public isConnected = AccountCtrl.get().isConnected
+  @state() public isConnected = false
```

Comment:
What are the implications of this change?

Reply:
I moved this down into constructor (as it is now coming from client, there was ssr error for undefined ethereum client in next)